src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const sevenDayLimit = 7;
+  const fiftyDollarOffer = 50;
+  const twentyDollarOffer = 20;
+  const threeDayLimit = 3;
```

Comment:
Let's don't hardcore the variable's name for exact amounts.
Just use abstract names like basicDiscount, bigDiscount etc. 
![image](https://user-images.githubusercontent.com/96409563/209931779-5d72d7f9-4d6a-4c2e-abb1-e2ee60322171.png)